src/js_native_api_v8.cc

```diff
@@ -191,10 +191,10 @@ inline static napi_status ConcludeDeferred(napi_env env,
 
 enum UnwrapAction { KeepWrap, RemoveWrap };
 
-inline static napi_status Unwrap(napi_env env,
```

Comment:
`static` for standalone functions in the .cc file should be OK - it should prevent any collisions with other compilation units. I believe Clang has a warning that asks to add `static` in such cases.
This is an opposed to the header files where the `static` keyword produces bad code by forcing each .cc file to have its own copy of that function.

Reply:
This function is located in an anonymous namespace. So the `static` mark is not needed here.

Reply:
Good point! I did not notice it. It seems that we have similar issue with other functions. Could we simplify this PR by implementing all such changes in another PR?